patches/minecraft/net/minecraft/advancements/AdvancementManager.java.patch

```diff
@@ -1,10 +1,10 @@
 --- ../src-base/minecraft/net/minecraft/advancements/AdvancementManager.java
 +++ ../src-work/minecraft/net/minecraft/advancements/AdvancementManager.java
-@@ -66,6 +66,7 @@
+@@ -65,6 +65,7 @@
+         this.field_193768_e = false;
          field_192784_c.func_192087_a();
          Map<ResourceLocation, Advancement.Builder> map = this.func_192781_c();
-         this.func_192777_a(map);
 +        this.field_193768_e |= net.minecraftforge.common.ForgeHooks.loadAdvancements(map);
+         this.func_192777_a(map);
```

Comment:
Overriding vanilla advancements is explicitly disallowed, hence why I put the event AFTER vanilla's load.

Reply:
Why is overriding vanilla advancements disallowed? Anyone can do it in the world files.